cmd/tbcd/tbcd.go

```diff
@@ -85,16 +85,22 @@ var (
 		},
 		"TBC_MAX_CACHED_KEYSTONES": config.Config{
 			Value:        &cfg.MaxCachedKeystones,
-			DefaultValue: int(1e6),
+			DefaultValue: int(2e6),
 			Help:         "maximum cached keystones during indexing",
 			Print:        config.PrintAll,
 		},
 		"TBC_MAX_CACHED_TXS": config.Config{
 			Value:        &cfg.MaxCachedTxs,
-			DefaultValue: int(1e6),
+			DefaultValue: int(2e6),
```

Comment:
Is there a reason to do this change in this pull request? Would we be able to do this in a separate PR with a separate changelog entry?